src/screens/benefit/details/converFun.tsx

```
@@ -0,0 +1,63 @@
+export function convertEligibilityFields(eligibility: any[]): JSONSchema7 {
+  const schema: JSONSchema7 = {
+    type: "object",
+    properties: {},
+  };
+
+  eligibility.forEach((eligibilityField: any) => {
+    schema.properties![eligibilityField.name] = {
+      type: "string",
+      title: eligibilityField.label,
+      enum:
+        eligibilityField.criteria.conditionValues instanceof Array
```

Biome

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)